packages/wrangler/src/r2/helpers.ts

```diff
@@ -48,19 +48,47 @@ export async function listR2Buckets(
 export async function createR2Bucket(
 	accountId: string,
 	bucketName: string,
-	jurisdiction?: string
+	jurisdiction?: string,
+	storageClass?: string
 ): Promise<void> {
 	const headers: HeadersInit = {};
 	if (jurisdiction !== undefined) {
 		headers["cf-r2-jurisdiction"] = jurisdiction;
 	}
+	if (storageClass !== undefined) {
+		headers["cf-r2-storage-class"] = storageClass;
+	}
 	return await fetchResult<void>(`/accounts/${accountId}/r2/buckets`, {
 		method: "POST",
 		body: JSON.stringify({ name: bucketName }),
 		headers,
 	});
 }
 
+/**
+ * Update the default storage class to `storageClass` of a bucket with the given `bucketName`
+ * within the account given by `accountId`.
+ */
+export async function putR2BucketStorageClass(
```

Comment:
`put...` seems like the wrong name here?
This is updating the storage class using a `PATCH` verb not `PUT`.
It feels like "put" is being used because it was copied over from when we actually "PUT" to the REST API?

Reply:
It's because of the naming convention in the R2 API itself (https://github.com/cloudflare/workers-sdk/pull/5429#discussion_r1546741564).